.idea/codeStyles/Project.xml

```diff
@@ -0,0 +1,27 @@
+<component name="ProjectCodeStyleConfiguration">
```

Comment:
.idea/codeStyles/Project.xml looks very similar to .idea/codeStyleSettings.xml, do we need both respictively duplicated code twice?

Reply:
good spot! I think this is due to the intellij 2017.3. Which changed the format of this.
I removed the one from the old version. So better update :)